hyperspy/tests/io/test_bcf.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import numpy as np
+
+from nose.tools import assert_true
+from hyperspy.io import load
+
+test_files = ['P45_instructively_packed_16bit_compressed.bcf',
+              'P45_12bit_packed_8bit.bcf',
+              'P45_the_default_job.bcf']  # for not compressed testing
+
+
+def test_load():
+    # test bcf from hyperspy load function level
+    # some of functions can be not covered
+    # it cant use cython parsing implementation, as it is not compiled
+    for thingy in test_files:
+        my_path = os.path.dirname(__file__)
+        filename = os.path.join(my_path, 'bcf_data', thingy)
+        print('testing bcf loading from hyperspy level...')
+        s = load(filename, downsample=2, cutoff_at_kV=10)
+        s = load(filename)
```

Comment:
Why load the same thing twice, but with different options? You also do not check that the loading result is correct - just that it completed and that the dtype is correct. 

I would pick a pixel and record it's non-zero elements here (in a numpy array) for reference, so that you can check if the correct values are loaded.

If not this, you could at least check that the shape of the loaded files is correct